servers/src/main/java/tachyon/web/WebInterfaceWorkerBlockInfoServlet.java

```diff
@@ -184,64 +184,46 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
   /***
    * Gets the {@link UIFileInfo} object based on file id.
    *
-   * @param tachyonFileSystem the {@link TachyonFileSystem} client
+   * @param fileSystem the {@link FileSystem} client
    * @param fileId the file id of the file
    * @return the {@link UIFileInfo} object of the file
    * @throws FileDoesNotExistException if the file does not exist
```

Comment:
Do you mind if we address this separately? There are several methods which would need update.